app/scripts/controllers/about.js

```diff
@@ -1,13 +1,4 @@
 angular.module('maktaba.controllers')
-  .controller('AboutCtrl', ['$scope', 'Utils', function($scope, Utils) {
-    $scope.openDialog = function(event) {
-      Utils.dialog('My name is a String', 'Hey there, I\'m a dialog', event,
-        function() {
-          console.log('Ok button has been clicked!');
-        });
-    };
+  .controller('AboutCtrl', ['$scope', , function($scope) {
```

Comment:
Empty array elements require elision=true.<br>'$scope' is defined but never used.